commands/monitoring.go

```diff
@@ -0,0 +1,350 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+)
+
+// Monitoring creates the monitoring commands hierarchy.
+func Monitoring() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "monitoring",
+			Short: "Display commands to manage monitoring",
```

Comment:
Do we want to label these commands as beta for now? 

```suggestion
			Short: "[Beta] Display commands to manage monitoring",
```

Adding `Hidden:  true,` to the top level command will hide it from the main list of commands if you want to further limit its discoverability for now.

Reply:
I think we can let it be in terms of hidden, since we should be entering open beta period next week. I will add the [Beta] marker though!